lib/msf/base/sessions/ssh_command_shell_reverse.rb

```diff
@@ -31,20 +31,19 @@ def desc
       'SSH command shell'
     end
 
-    def shell_command(cmd)
+    def shell_command(cmd, timeout = 5)
```

Comment:
The `shell_command` API was changed when session verification was added:

https://github.com/rapid7/metasploit-framework/commit/69c525c2f7eca4c16461b765d08b57be908456e8

Previous stack trace:
```
[08/04/2022 13:49:05] [e(0)] core: Error in stream server server monitor: wrong number of arguments (given 2, expected 1)

Call stack:
/Users/user/Documents/code/metasploit-framework/lib/msf/base/sessions/ssh_command_shell.rb:36:in `shell_command'
/Users/user/Documents/code/metasploit-framework/lib/msf/base/sessions/command_shell.rb:96:in `bootstrap'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/handler.rb:271:in `register_session'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/handler/reverse_ssh.rb:141:in `create_session'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/handler/reverse_ssh.rb:121:in `block in init_fd_client'
/Users/user/.rvm/rubies/ruby-2.7.2/lib/ruby/2.7.0/timeout.rb:95:in `block in timeout'
/Users/user/.rvm/rubies/ruby-2.7.2/lib/ruby/2.7.0/timeout.rb:33:in `block in catch'
/Users/user/.rvm/rubies/ruby-2.7.2/lib/ruby/2.7.0/timeout.rb:33:in `catch'
/Users/user/.rvm/rubies/ruby-2.7.2/lib/ruby/2.7.0/timeout.rb:33:in `catch'
/Users/user/.rvm/rubies/ruby-2.7.2/lib/ruby/2.7.0/timeout.rb:110:in `timeout'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/handler/reverse_ssh.rb:115:in `init_fd_client'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/handler/reverse_ssh.rb:92:in `block in setup_handler'
/Users/user/Documents/code/metasploit-framework/lib/rex/proto/ssh/server.rb:145:in `on_client_connect'
/Users/user/Documents/code/metasploit-framework/lib/rex/proto/ssh/server.rb:94:in `block in start'
/Users/user/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/rex-core-0.1.16/lib/rex/io/stream_server.rb:38:in `on_client_connect'
/Users/user/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/rex-core-0.1.16/lib/rex/io/stream_server.rb:165:in `monitor_listener'
/Users/user/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/rex-core-0.1.16/lib/rex/io/stream_server.rb:70:in `block in start'
/Users/user/Documents/code/metasploit-framework/lib/rex/thread_factory.rb:22:in `block in spawn'
/Users/user/Documents/code/metasploit-framework/lib/msf/core/thread_manager.rb:105:in `block in spawn'
```

Reply:
I haven't investigated why Meterpreter base sessions doesn't crash, which has the old API and implementation still:

https://github.com/rapid7/metasploit-framework/blob/8febcd1b1337edd151c877b4a72684460963029a/lib/msf/base/sessions/meterpreter.rb#L262-L280